src/services/magic/TensorflowWorker.ts

```diff
@@ -34,6 +34,8 @@ tf.loadLayersModel(modelUrl).then((model) => {
   }
 
   logger.debug('Model initialized!', model)
+  const shape = model.inputs[0].shape.slice(1)
+  if (!shape) model.call(tf.zeros([1, ...shape]), {})
```

Comment:
We should have a mechanism for awaiting the end of the initialization in context of implementing proper page loading screen in the future. Perhaps reverting to AsyncInitialized pattern is in place.